modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -6,7 +6,12 @@ endif::[]
 
 image::prepare-to-install-breadcrumb.png[]
 
-Before you download and install Anypoint Service Mesh, you must meet the  software, licensing, and permissions and roles requirements describes in this section.
+Before you download and install Anypoint Service Mesh, make sure that you review and fulfill the software, licensing, and permissions and roles requirements describes in this section. 
+
+== About the Audience
+
+Anypoint Service Mesh is a enterprise microservices management application. Microservices are deployed in containerized environments to achieve best business results. Anypoint Service Mesh is developed using Istio and Kubernetes. Therefore, you must have basic working knowledge of Istio and Kubernetes to be able to install and use (derive benefit from??) Anypoint Service Mesh.
```

Comment:
>(derive benefit from??)

We need to remove this?

Reply:
I would just keep this much 
"Anypoint Service Mesh is developed using Istio and Kubernetes. Therefore, you must have a basic working knowledge of Istio and Kubernetes to be able to install and use Anypoint Service Mesh"

Reply:
I would just put this much "you must have a basic working knowledge of Istio and Kubernetes to be able to install and use Anypoint Service Mesh"

Reply:
Done! Thanks!